packages/esm-patient-orders-app/src/api/api.ts

```diff
@@ -5,10 +5,11 @@ import {
   openmrsFetch,
   type OpenmrsResource,
   parseDate,
-  type Visit,
   restBaseUrl,
+  type Visit,
 } from '@openmrs/esm-framework';
-import { type OrderPost, useVisitOrOfflineVisit, useSystemVisitSetting } from '@openmrs/esm-patient-common-lib';
+import { type OrderPost, useSystemVisitSetting, useVisitOrOfflineVisit } from '@openmrs/esm-patient-common-lib';
+import { orderBasketStore, type OrderBasketStore } from '@openmrs/esm-patient-common-lib/src/orders/store';
```

Comment:
Is it possible to use a single import statement?

Reply:
You'll likely need to make sure that those exports are re-exported from https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-common-lib/src/orders/index.ts first. That might need to happen in a separate PR.

Reply:
As discussed with @brandones , moving the implementation of saving the encounter and orders to the `common-lib` seemed like a better idea. Therefore, I have created the `postOrdersOnNewEncounter` function in the [postOrders.ts](https://github.com/usamaidrsk/openmrs-esm-patient-chart/blob/f5e5969471386920761f3b1dbae443c377d2a297/packages/esm-patient-common-lib/src/orders/postOrders.ts#L6) file.

cc @ibacher @denniskigen @samuelmale